egov-persister/auth-sek-session-persister.yml

```
@@ -49,3 +49,22 @@ serviceMaps:
+          - query: INSERT INTO treasury_head_breakup_data (consumer_code, head_mapping,calculation, tenant_id, createdtime) values(?, ?, ?, ?, ?)
+            basePath: $
+            jsonMaps:
+              - jsonPath: $.consumerCode
+              - jsonPath: $.headAmountMapping
+                type: JSON
+                dbType: JSONB
+              - jsonPath: $.calculation
+                type: JSON
+                dbType: JSONB
+              - jsonPath: $.tenantId
+              - jsonPath: $.createdTime
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)